helm-chart/mezod/values.yaml

```diff
@@ -2,6 +2,9 @@ image: "mezo/mezod"
 tag: "v1.0.0"
 imagePullPolicy: Always
 
+# -- Select whether you want to run a validator/seed node/RPC node ("validator"/"seed"/"rpc"/"rpc-archive")
```

Comment:
There are 4 options, but `rpc` isn't used anywhere. Let's remove `rpc-archive` and use `rpc` for that.
RPC should have pruning=nothing